pkgs/by-name/do/dove/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchzip,
+  stdenvNoCC,
+  nix-update-script,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "dove";
+  version = "2025.02.01.1";
+
+  src = fetchzip {
+    url = "https://codeberg.org/celenity/Dove/raw/tag/${finalAttrs.version}/archives/dove.zip";
+    hash = "sha256-tAIWO/I9k/TE4jyGS3vHsehZFszJp03ay/tcK+PVnLI=";
+    stripRoot = false;
+    # general.config.filename is hard-coded to mozilla.cfg in wrapFirefox, so removing this setting:
+    postFetch = ''
+      sed -i '/general.config.filename/d' $out/dove.cfg $out/prefs/dove.js
+    '';
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out
+    cp -r $src/policies.json $src/dove.cfg $src/prefs $out/
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "A suite of configurations & advanced modifications for Mozilla Thunderbird with a focus on privacy, security, freedom, & usability";
+    homepage = "https://dove.celenity.dev/";
+    license = lib.licenses.gpl3Plus;
```

Comment:
Why is this gpl3Plus instead of gpl3Only?

Reply:
Thoward the end of the license there is is [this mention](https://codeberg.org/celenity/Dove/src/branch/pages/LICENSE#L214):

>    Phoenix
>    Copyright (C) 2024  Magnesium1062
>
>    This program is free software: you can redistribute it and/or modify it under the terms of the GNU General  Public License as published by the Free Software Foundation, either version 3 of the License, **or (at your option) any later version**.

Reply:
After the

> END OF TERMS AND CONDITIONS

line? The GPL license itself should not be modified as this. The license header should be added in the source code instead.

Reply:
yeah, it's probably not orthodox, for me the intent is GPL3+ but I'm not a lawyer. I can open an issue to get them add proper headers.

Reply:
Will be fixed in the release.

Reply:
Fixed!